routes/matches/matches.js

```diff
@@ -822,7 +862,7 @@ router.get("/limit/:limiter", async (req, res, next) => {
  */
 router.get("/:match_id/config", async (req, res, next) => {
   try {
-    let sql = "SELECT * FROM `match` WHERE id = ?";
+    let sql = "SELECT mtch.* FROM `match` mtch WHERE mtch.id = ?";
```

Comment:
Does this really need to be changed if you haven't made any changes to it aside from adding on the identifier when it's not needed?